### PR TITLE
Prepare release candidate v0.9.0-RC2

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

File: assembly/pom.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <parent>
     <groupId>org.tachyonproject</groupId>
     <artifactId>tachyon-parent</artifactId>
-    <version>0.9.0-RC1</version>
+    <version>0.9.0-RC2</version>
```
Comment (aaudiber) [inline assembly/pom.xml]: would it make sense for 0.9.0-RC2 to be stored in a variable in the parent pom?

Comment (apc999) [inline assembly/pom.xml]: i guess without specifying the parent version, it can not inherit a variable from parent pom.

**Comment by aaudiber** [comment]
Jenkins, test this please

File: assembly/pom.xml (hunk 1 above)
Comment (aaudiber) [inline assembly/pom.xml]: got it

**Comment by aaudiber** [comment]
LGTM

**Pull request merged**